ping/ping.c

```diff
@@ -449,9 +449,15 @@ main(int argc, char **argv)
 			break;
 		case 'p':
 			rts.opt_pingfilled = 1;
+			if (outpack_fill) {
+				free(outpack_fill);
+				outpack_fill = NULL;
```

Comment:
I do not think that we have to clear the `outpack_fill` here, we assign it new value right after this. Also `free(NULL)` is no-op so this whole part can be just:

```
        ...
        rts.opt_pingfilled = 1;
        free(outpack_fill);
        outpack_fill = strdup(optarg);
        if (!outpack_fill)
                ...
```